bucket/rabbitmqadmin-ng.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "2.2.1",
+    "description": "A command line tool for RabbitMQ that uses the HTTP API",
+    "homepage": "https://github.com/rabbitmq/rabbitmqadmin-ng",
+    "license": "MPL-2.0",
```

```suggestion
    "license": "Apache-2.0|MIT",
```

As per GH repo details.

---

Good catch!